blog/_posts/2021-01-19-scala-native-0.4-release.md

```diff
@@ -0,0 +1,225 @@
+We will start with changes according to Scala Native compiler plugin, especially supported Scala versions and cross-platform testing.
+Next we will talk about changes to C interoperability and SBT build that need to be taken into account since when working this release.
+Last, but not least we will discuss why we needed to break binary compatibility with previous versions.
+
+## Support for latest Scala 2 versions
+Until now Scala Native supported only Scala 2.11, which is now perceived by most of the developers as deprecated.
+We can proudly announce that Scala Native is now compatible with Scala versions 2.12.13 and 2.13.4, in addition to the 2.11.12 with known bugs fixed.
+
+## Reflective instantiation
+The SN plugin removes unused definitions of classes, methods, and variables during the linking stage to reduce the size of the output binary.
+This used to make using reflective calls very difficult. Fortunately, this is a thing of the past---with release 0.4.0 we
+introduced a subset of reflective operations allowed to be used.
+
+It would not be wise to enable reflective calls to all definitions as we would need to load many more objects and increase the size of the binary.
+That’s why you need to signal to the compiler which classes can use this mechanism - it can be done by marking your `trait`, `class`, or `object`
+with the `@EnableReflectiveInstantation` annotation. It will also make all of its descendants able to be reflectively instantiated.
+
+Annotated classes and modules, having a concrete implementation, can be accessed via the provided `scalanative.reflect.Reflect` API.
+If you have used Scala.js before, it may seem similar to you, as the new implementation uses exactly the same API.
+
+## Cross-platform testing using JUnit
+Until recently, all our internal tests were executed using our own native testing framework, mainly due to lack of proper
+reflection support in Scala Native. After getting rid of this blocker, we have decided to use JUnit instead and in 
+result implemented the native plugin for this framework. Thanks to this change you are also able to also use it in your own projects.  
+Naturally you are still able to use any other testing framework, as long as it supports Scala Native.
+
+To enable Native JUnit plugin tests, all you need to do is to add the two following lines to your `build.sbt`.
+```scala
+addCompilerPlugin("org.scala-native" % "junit-plugin" % nativeVersion cross CrossVersion.full)
+libraryDependencies += "org.scala-native" %%% "junit-runtime" % nativeVersion % "test"
+```
+
+The above settings, when running `sbt test`, would result in the tests being compiled using the SN plugin and executed 
+with a native implementation of the JUnit framework. What is more you can easily run your tests both on JVM and Native 
+with a simple adjustment to your build, you can find guide how to do this in [Scala Native reference](https://scala-native.readthedocs.io/en/latest/)
```

Comment:
"guide" should either be "guidance" or "a guide". also the comma should either be a period or a semicolon.